### ISSUE TITLE
Animation for collapsible only animates some times for regular box, never for sidebar.

### ISSUE THREAD

**Issue by mrmagcore** [grommet/grommet#6434]
### Expected Behavior

Smooth animation should happen on all collapsible elements.

### Actual Behavior

On plain box element, animation sometimes works, sometimes there is no animation.  On the sidebar, there is no animation at all.

### URL, screen shot, or Codepen exhibiting the issue

https://dq2r7c.csb.app/

### Steps to Reproduce

follow the github grommet tutorial

### Your Environment

react 18.2.0
grommet 2.27.0

chrome Version 106.0.5249.91 (Official Build) (64-bit)
or 
firefox 106.0
OS: 18.04.1-Ubuntu SMP Thu Sep 1 16:25:16 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

**Comment by jcfilben** [comment]
Hi @mrmagcore you can get the sidebar to animate by removing the `this.state.showSidebar` check. The `open` prop on Collapsible will handle hiding/showing the content within the collapsible container. I modified your codesandbox to show this change: https://codesandbox.io/s/grommet-v2-template-forked-nff36n?file=/index.js.

The Github Grommet starter app is a bit out of date, I think we need to go back and update this to resolve issue like this and make sure we are demonstrating the latest recommendations and components.

**Comment by mrmagcore** [comment]
Thanks, that's very nice of you to point out, I just didn't read the diff markup in the tutorial correctly.
I'd love it if the "copy this" on the tutorial would selectively copy just the new code without the "+" or copy the final version of the code after the diff has been applied.

**Comment by mrmagcore** [comment]
It's an edge case, but if you look at the smaller collapsible in my sandbox, you will see that it usually, but not always, animates.  That's a little weird.

**Comment by jcfilben** [comment]
> I'd love it if the "copy this" on the tutorial would selectively copy just the new code without the "+" or copy the final version of the code after the diff has been applied.

Yeah I think the information could be presented in a better format to make it a little more clear and easier to copy over.

> It's an edge case, but if you look at the smaller collapsible in my sandbox, you will see that it usually, but not always, animates. That's a little weird.

I saw you mentioned this in the original issue and I was trying to replicate it, but I wasn't able to see this issue even after opening and closing a bunch of times
